plugins/modules/zabbix_map.py

```diff
@@ -160,9 +159,6 @@
 ### Create Zabbix map "Demo Map" made of template 'map.j2'
 - name: Create Zabbix map
   community.zabbix.zabbix_map:
-    server_url: http://zabbix.example.com
```

Comment:
doesn't this module miss the: `zabbix_utils.require_creds_params(module)` call?

Reply:
Good catch, thank you. Adding.